docker/Makefile

```diff
@@ -66,11 +60,11 @@ pdo-dev-image:
 	@echo "Building pdo-dev docker image"
 	docker build $(DOCKER_BUILD_OPTS) -f Dockerfile.pdo-dev -t pdo-dev .
 
-pdo-composition: pdo-dev-image
-	@echo "Running PDO docker composition for Sawtooth"
-	env PDO_REPO_BRANCH=$$(git rev-parse --abbrev-ref HEAD) $(DOCKER_COMPOSE_COMMAND) $(DOCKER_COMPOSE_OPTS_STL) build
+pdo-composition-ccf: pdo-dev-image
+	@echo "Running PDO docker composition for CCF"
+	env PDO_REPO_BRANCH=$$(git rev-parse --abbrev-ref HEAD) $(DOCKER_COMPOSE_COMMAND) $(DOCKER_COMPOSE_OPTS_CCF) build
 	# Note:
```

Comment:
why is this parameterized for CCF? its just building PDO. there should be nothing in here that is specific to ccf. That should be a completely runtime determination.

Reply:
We can remove this parameterization now.

Reply:
Done

Reply:
Sorry for being two days late, parametrization for ccf, if being removed, must be removed everywhere. We still use ccf as part opf docker compose file names. and pdo ledger type is omnipresent. Given that we want the ledger abstraction to be still present even after removing Sawtooth, I would either make sure that logic checks for ledger type or for now explicitly calls our ccf. We do not want to give the impression that a line of code (that does somethng with eldger) will apply as is without change to  another choice of ledger (unless we know that it will)